detection_rules/etc/version.lock.json

```diff
@@ -8161,25 +8175,25 @@
       }
     },
     "rule_name": "Windows Script Interpreter Executing Process via WMI",
-    "sha256": "4fd150bb0360d26c6447ab45a3237e6572e59b7378963e569c9be82fcc773919",
+    "sha256": "6e831fce582191305c1d7d3da75c0f080265f7c68e86194ad2a7f6b5bc6e4bad",
     "type": "eql",
-    "version": 101
+    "version": 102
   },
   "b6dce542-2b75-4ffb-b7d6-38787298ba9d": {
     "min_stack_version": "8.3",
     "previous": {
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "Azure Event Hub Authorization Rule Created or Updated",
-        "sha256": "570457480dcddf764559deab4f29056049caf7e7b1cb98c2f902ec08e1d645c1",
+        "sha256": "cda9d6420803eeff9b35d9028aa6935ff4d213c1caa90595097960c9e1acd8bb",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.